### ISSUE TITLE
Add listing page to front-end for categories

### ISSUE THREAD

**Issue by asmecher** [pkp/pkp-lib#4524]

**Comment by mfelczak** [comment]
Thanks @asmecher. Looks like we're just missing the page template.

**Comment by asmecher** [comment]
Whoops! Sorry, @mfelczak: https://github.com/asmecher/ojs/commit/461e1b666a824b7d173d434b98df3610e266d5b7

**Comment by mfelczak** [comment]
Thanks @asmecher. One immediate note when testing in a 3.1.1-4 install: the page title looks to have an untranslated locale key.

**Comment by asmecher** [comment]
@mfelczak, can you include a screenshot or identify the locale key?

**Comment by mfelczak** [comment]
Hi @asmecher, this was the page title locale key in an OJS 3.1.1-4 install, i.e. from the template header:

```{include file="frontend/components/header.tpl" pageTitle="category.categories"}```

**Comment by NateWr** [comment]
@mfelczak this _may_ have been fixed in https://github.com/pkp/pkp-lib/issues/4446 which went into 3.1.2. The relevant commit was https://github.com/pkp/pkp-lib/pull/4454/commits/d1209076973c118b89ff60926bc6814d3c4b917c. Can you check if that helps?

**Comment by asmecher** [comment]
@mfelczak, just a heads-up that this work remains uncommitted pending some testing.

**Comment by mfelczak** [comment]
Hi @asmecher and @NateWr, sorry I missed your original msg. I don't have a working dev environment at the moment but @ewhanson may be able to confirm. @asmecher, should we test the original commits against `main` at this point?

**Comment by asmecher** [comment]
Hmm, no, I think at this point it'll need some redevelopment before it'll test against `main`. Is this still a hosting priority?

**Comment by mfelczak** [comment]
Yes, we've continued to receive interest about this from journals with many categories that don't display well in the sidebar as a long list. Would it be helpful to lend a hand with the porting to `main` at this point?

**Comment by israelcefrin** [comment]
Hello @asmecher , this is an old issue and no comments has been added since 2021. I am reviewing all issues pending in the "Reader Interface" project to move them to the Accessibility Roadmap. I will close this issue for now. But, let me know if we need to keep it open.

**Comment by asmecher** [comment]
That's fine with me, @israelcefrin; this has already been implemented with https://github.com/pkp/pkp-lib/issues/9295.
